test/binary/linking-section.txt

```diff
@@ -23,10 +23,10 @@ section("linking") {
     count[2]
     str("data1")
     p2align[2]
-    flags[0]
+    flags[1]
```

Comment:
wow is this file basically assembly for wabt?

Reply:
Kind of.. more like a binaryen file constructor... kinda cool right?